.github/actions/spelling/expect.txt

```diff
@@ -0,0 +1,493 @@
+Abf
+Abh
+acbf
+addressparser
+AHyperlink
+alekstorm
+algo
+allcaps
+amd
+amlog
+anonymize
+Aopen
+api
+applypatch
+apps
+AQws
+args
+argv
+ARIo
+Arnfj
+asciidoc
+attrs
+autocrlf
+autolearn
+automerge
+autosquash
+avarab
+aws
+backend
+basedon
+BAYES
+bbox
+BCAE
+bcbb
+bfbde
+BFk
+bigfilethreshold
+bigint
+Bjarmason
+BKBc
+BKBd
+BKL
+blockquote
+blocksource
+BNew
+bordercolor
+borderopacity
+brk
+BTr
+buf
+bugfix
+busybox
+BXx
+CABPp
+cacheinfo
+callbackfn
+callchain
+CBGFIg
+ccd
+CDcd
+CEu
+charset
+chdir
+CIimap
+CIPh
+CIsmtp
+CIsmtphost
+CIsmtpopts
+CIsmtppass
```

Comment:
this set should look familiar

Reply:
I really don't like that there are _two_ lists with essentially the same content... :-(

Reply:
Yeah, I get it.

Adding a plug-in for VSCode is on the list of things I might do: https://github.com/check-spelling/check-spelling/wiki/Feature%3A-Visual-Studio-Code-plugin, but it's quite a ways down and I haven't started thinking through how to actually do it.

Fwiw, in this case, cSpell is set to think about these items as regular expressions. I have no idea someone chose to do that. You can see that I'm just treating them as word-like terms.

Reply:
For the record, I have no intention replacing cSpell in my VS Code installation with any other spell checker. I would want to use the same word list, not another spell checker.